roles/etcd/tasks/certificates/fetch_server_certificates_from_ca.yml

```diff
@@ -145,6 +145,8 @@
   unarchive:
     src: "/tmp/{{ inventory_hostname }}/{{ etcd_generated_certs_dir }}/{{ etcd_cert_subdir }}.tgz"
     dest: "{{ etcd_cert_config_dir }}"
+    owner: etcd
```

Comment:
Why is it necessary? Tarball should preserve permissions set on `etcd_ca_host`

Reply:
You tell me ;)
Without this it restores file owner/group with `root:root` which makes etcd fail to read it's configuration.

Reply:
Does `etcd_ca_host` has these set with `root:root`? What's you config - standalone etcd or static pods?

Reply:
@vrutkovs: I tried manually set `etcd:etcd` but Ansible later sets it to `root:root` anyway with this task.